virttest/qemu_vm.py

```diff
@@ -1304,7 +1304,7 @@ def add_virtio_option(option, value, devices, dev, dev_type):
             if option in options:
                 dev.set_param(option, value)
 
-        def _get_pci_bus(devices, params, dtype=None, pcie=False):
+        def _get_pci_bus(devices, params, dtype=None, pcie=False, root_bus=False):
```

Comment:
Basically the `root_bus` is a symlink to `{'aobject': params.get('pci_bus', 'pci.0')}` right?

Reply:
@ldoktor  thanks your comments, this patch is closed.
I have new pull request https://github.com/avocado-framework/avocado-vt/pull/2387, would you like to help review it.

Thanks,
Xu